Appraisals

```diff
@@ -1,19 +1,4 @@
-if ['2.2.2', '2.3.3', '2.6.3'].include?(RUBY_VERSION)
-  appraise 'rails506' do
-    gem 'rails', '5.0.6'
-    gem 'sqlite3', '~> 1.3.6'
-  end
-
-  appraise 'rails521' do
-    gem 'rails', '5.2.1'
-    gem 'sqlite3', '~> 1.3.6'
-  end
-
-  appraise 'rails522' do
-    gem 'rails', '5.2.2'
-    gem 'sqlite3', '~> 1.3.6'
-  end
-
+if ['3.2.3'].include?(RUBY_VERSION)
```

Comment:
I think for open source gems, we should aspire to be backwards-compatible for the sake of the general public.
Hence, having multiple Rubies appraised would be a net benefit.
As for the Rails version, I think it should be whatever combos of Ruby and Rails that are officially supported.

Reply:
Okay. I can do that.

Reply:
FYI this is dead config since we don't use Travis CI any more: https://github.com/appfolio/inheritable_fixtures/blob/master/.travis.yml
Replacement should ideally be CCI, but I suppose local testing works OK.